{"uuid": "94aff8db-f049-4efb-ae71-42577e319ab6", "children": ["0baf5901-949b-41a7-8077-93520f7cd289"], "befores": [{"name": "setup_teardown", "status": "broken", "statusDetails": {"message": "AttributeError: 'function' object has no attribute 'quit'\n", "trace": "  File \"D:\\Python39\\lib\\site-packages\\pluggy\\_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"D:\\Python39\\lib\\site-packages\\_pytest\\setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n  File \"D:\\Python39\\lib\\site-packages\\pluggy\\_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"D:\\Python39\\lib\\site-packages\\_pytest\\fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"D:\\Python39\\lib\\site-packages\\_pytest\\fixtures.py\", line 891, in call_fixture_func\n    fixture_result = next(generator)\n  File \"E:\\AutoTest\\FgAIHelp\\tests\\base_test.py\", line 58, in setup_teardown\n    self.driver.quit()\n"}, "start": 1744361664317, "stop": 1744361668466}], "start": 1744361664317, "stop": 1744361668570}